### ISSUE TITLE
Allow forwarding to different/secondary email address

### ISSUE THREAD

**Issue by neubland** [mozilla/fx-private-relay#564]
This is a Feature Request.

Right now, mails are automatically relayed to the primary Firefox Account mail. This is good enough for many mail configurations. For more complex mail setups, it would be nice to relay mails to a different account. It could be that one has multiple Inboxes/Addresses and all the potential spam and junk is not supposed to go the same email that is used to manage the Firefox Account.

Proposed solution: Allow setting a specific email for Relay, different from the main one. Alternative: Firefox Account already supports "Secondary email", this could be used.

AnonAddy has this feature.

**Comment by groovecoder** [comment]
There are a couple AMO reviews requesting this too.

**Comment by jeroen7s** [comment]
Same here, the email that is linked to my Firefox Account is no longer my primary email (hasn't been for many years) so the ability to have another email-address to forward to would definitely be useful.

**Comment by tedschuh** [comment]
Consider as a future enhancement.

**Comment by Oryoro** [comment]
It would also be nice if you could have multiple email addresses that you can forward to instead of just one.

**Comment by ridicolos** [comment]
Would love to see this be implemented.

**Comment by mark-omarov** [comment]
I have multiple email addresses I need to be able to relay to, since I'm now migrating to Firefox I figured that would be great to use Firefox Relay, but unless this gets implemented I honestly don't get any value from it.
I'm willing to work on the FE part, since BE in Python I can't help much.
Will keep an eye on this issue and feel free to mention me if it's something I can contribute for. Cheers!

**Comment by imcardle** [comment]
I too would like to see this new feature added so that emails can be forwarded to alternative (or multiple) real email addresses.

**Comment by luisnabais** [comment]
Any news on this request? Please take the users opinions into consideration...

**Comment by dogmax** [comment]
Just chiming in to say that I also request this feature.

**Comment by morganbanet** [comment]
I've been thinking about this lately and am glad people have already bought this up. I have multiple folders in my inbox and it would be easier to maintain and organize them if Firefox Relay offered more than one email recepient for each mask.

**Comment by covelloz** [comment]
Paid for Relay, realized you couldn't just add an e-mail address to forward to like you can with SimpleLogin. Only can create masks that forward to the e-mail of your FF account. Cancelled immediately. Please consider adding this feature.

**Comment by lpcvoid** [comment]
Yeah, I was also sad to see this after paying for a year. Please consider adding! Or at least, make an option which can take the secondary email in your account too - then I would relay all the trash emails to my trash account, instead of my main email that I want to get Mozilla updates on.

For now, I set my trash email as my primary email in order to properly use Relay, but I would love to not have to do this.

**Comment by edizarca** [comment]
I'd like to migrate from the email account I used when signing up to mozilla, relay being fixed on primary mozilla account email address prevents me from using it with my new email address. I'd like this feature very much also.

**Comment by Vinnl** [comment]
@edizarca If you change the email address for your Mozilla Account then Relay _should_ pick that up. If it doesn't, [see here](https://connect.mozilla.org/t5/ideas/change-firefox-relay-email-address/idc-p/45113/highlight/true#M26077).

**Comment by EMMLynch** [comment]
This issue is being automatically closed as it is older than 2 years and has not been addressed. If this issue is still relevant, please feel free to reopen it with updated information.

**Comment by lpcvoid** [comment]
At least mark it as WONTDO instead of completed, if there's no plans to ever introduce this.

**Comment by hbirring01** [comment]
I would love this feature

**Comment by nstrelow** [comment]
I think this is still a relevant feature request, whenever there is time to build it ^^

**Comment by robin-paulson** [comment]
I changed my primary email address in my Firefox account, but Relay emails are still forwarded to my now defunct primary, rather than the new address I put there. I was surprised the two are not linked.

**Comment by robin-paulson** [comment]
Also, this is not completed. It's not OK to close and ignore
